### DISCUSSION TITLE
zigbee2mqtt LXC remote access ?

### DISCUSSION THREAD

**Discussion by jolypas** [tteck/Proxmox#3115] [Help Needed] [answered]
Hello

I installed zigbee2mqtt using the script. It works fine but I would like to be able to remote access to make some backup.

I have automatic login for root on proxmox console.
How to scp or ssh ?
What is the default password for root if any ? or should have to create one ?

Thks

**Comment by thijspomper** [comment, marked as answer]
Hi,

I'd suggest setting a root password with "passwd".
Next step would be to edit /etc/ssh/sshd_config and change "#PermitRootLogin prohibit-password" with "PermitRootLogin yes" and restart

**Reply by jolypas** [reply]
Thanks a lot
That what I'll do !
